README.md

```diff
@@ -397,8 +397,8 @@ this list aims to be broader and include apps from various other ecosystems in v
 
 ### Network Monitoring
 
-- [Hotwire](https://github.com/emmanueltouzery/hotwire) - GUI that leverages the wireshark and tshark infrastructure to capture traffic and explore the contents of tcpdump files `#rust`.
 - [Echo](https://github.com/lo2dev/Echo) - Simple utility to ping websites using GTK4 and Libadwaita. `#python` `#libadwaita`.
```

Comment:
```suggestion
- [Echo](https://github.com/lo2dev/Echo) - Simple utility to ping websites `#python` `#libadwaita`.
```

No need to specify GTK 4 and `libadwaita` in the description, the tags are here for that.
Also, I would recommend using the Flatpak links (instead of GitHub). Since it has more screenshots, I think it makes a better promotion of the application.